liam2/alignment.py

```diff
@@ -340,6 +340,7 @@ def compute(self, context, score, need=None,
             # calculate without leave_filter and without take_filter
             if need is None:
                 need = sum(score)
+            need = np.floor(need).astype(int)

```

Comment:
I can only deal with this set up as far as i understand the implementation right now.
@AlexisEidelman any comments ?